modules/users/utils/pfp.ts

```diff
@@ -0,0 +1,27 @@
+import { ModuleContext } from "../_gen/mod.ts";
+import { User } from "./types.ts";
+
+const EXPIRY_SECS = 60 * 60 * 24; // 1 day
+
+export const withPfpUrl = async <T extends ModuleContext>(
+    ctx: T,
+    user: Omit<User, "profilePictureUrl">,
+    uploadId: string | null,
+) => {
+    if (!uploadId) {
+        return {
+            ...user,
+            profilePictureUrl: null,
+        };
+    }
+
+    const { files: [{ url }] } = await ctx.modules.uploads.getPublicFileUrls({
```

Comment:
should run in batch taking multiple users & upload ids to make one request all at once